File: app/src/main/java/com/canli/oya/traininventory/utils/DataBindingUtils.java
package com.canli.oya.traininventory.utils;

import android.text.TextUtils;

public class DataBindingUtils {

    public static String[] splitLocation(String location) {
        return TextUtils.isEmpty(location) ? null : location.split("-");
    }

    public static String encloseInParanthesis(String category){
        return "(" + category + ")";
    }
}

File: app/src/main/java/com/canli/oya/traininventoryroom/utils/DataBindingUtils.kt
@file:JvmName("DataBindingUtils")

package com.canli.oya.traininventoryroom.utils

fun splitLocation(location: String?): List<String>? {
    return if (location.isNullOrBlank()) null
            else location.split("-")
}

fun encloseInParanthesis(category: String): String {
    return "($category)"
}